shared/scripts/setup-ssh-user.sh

```diff
@@ -73,7 +73,7 @@ echo "Set password"
 echo "${USER}:${PASSWORD}" | sudo chpasswd
 
 echo "Add /etc/sudoers.d/${USER}"
-sudo tee "/etc/sudoers.d/${USER}" > /dev/null <<"EOF"
```

Comment:
Odd that this worked before, looks like it would be putting `${USER}` into the `/etc/sudoers.d/${USER}` file as parameter substitution is turned off with the double quotes.

Reply:
lol this was luck 😄